ci-operator/config/redhat-developer/jenkins-operator/redhat-developer-jenkins-operator-master.yaml

```diff
@@ -0,0 +1,46 @@
+base_images:
+  base:
+    name: ubi-minimal
+    namespace: ocp
+    tag: "8"
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.12
+images:
+- dockerfile_path: Dockerfile
+  from: base
+  inputs:
+    src:
+      paths:
+        - destination_dir: .
+          source_path: /go/src/github.com/redhat-developer/jenkins-operator/.
+  to: jenkins-operator
+#promotion:
+#  name: "4.2"
+#  namespace: ocp
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 200m
+      memory: 200Mi
+tag_specification:
+  name: "4.3"
+  namespace: ocp
+#tests:
+#- as: unit
+#  commands: make test
```

Comment:
Could you add an empty `make test` to your repo and them un-comment his? The idea is to keep test steps here as high-level as possible and then iterate on those make targets in the jenkins repo.

Reply:
( could be a separate PR too )

Reply:
I just enabled the tests, was waiting for the tests to pass before I added any.